i18n/en.pot

```diff
@@ -101,12 +101,6 @@ msgstr "Data set"
 msgid "Organisation units"
 msgstr "Organisation units"
 
-msgid "Start date"
-msgstr "Start date"
-
-msgid "End date"
-msgstr "End date"
-
```

Comment:
@kabaros this change in extracted i18n looks incorrect... the `i18n.t('Start date')` call was moved but should still be picked up by the i18n scanner, so it looks like there might be a problem here... I think this means that those strings won't be translatable after this change?

Reply:
Ah, I see now this was changed to `Start Date` (capital D), which was already used somewhere else.  Our [design system indicates](https://ui.dhis2.nu/patterns/writing) that we should use sentence case rather than headline case, so I think we should change this to `Start date` in all instances.  That can be a different change (on master with backport) as it's a minor detail, but does affect what translations will be included in the final app bundle.  cc @cooper-joe